epub33/overview/index.html

```diff
@@ -127,6 +169,10 @@ <h2>Package Document</h2>
 					preview content, or assemble an index or dictionary from its constituent XHTML Content
 					Documents.</p>
 
+				<p class=ednote>
+					The <code>collection</code> element is rarely used in practice; is it worth keeping its reference in the overview document?
```

Comment:
I would leave it since the section is defining the major sections of the package document. It might be useful to turn this into a proper note and give some context, though. Maybe:

> The `collection` element is not currently used in any specifications that are actively maintained by the EPUB 3 Working Group. Its future use in authoring EPUB Publications is uncertain.